src/compas_rhino/artists/capsuleartist.py

```diff
@@ -39,10 +70,19 @@ def draw(self, u=10, v=10):
         """
         vertices, faces = self.shape.to_vertices_and_faces(u=u, v=v)
         vertices = [list(vertex) for vertex in vertices]
-        points = [{'pos': point, 'color': self.color} for point in vertices]
-        polygons = [{'points': [vertices[index] for index in face], 'color': self.color} for face in faces]
-        guids = compas_rhino.draw_points(points, layer=self.layer, clear=False, redraw=False)
-        guids += compas_rhino.draw_faces(polygons, layer=self.layer, clear=False, redraw=False)
+        guids = []
+        if show_vertices:
+            points = [{'pos': point, 'color': self.color} for point in vertices]
+            guids += compas_rhino.draw_points(points, layer=self.layer, clear=False, redraw=False)
+        if show_edges:
+            raise NotImplementedError
```

Comment:
i have left this open for now. the capsule doesn't actually have any edges. they could be generated to be able to visualize the wireframe. in general, the question is what the most relevant information is. i would argue that even showing the vertices is questionable...

Reply:
Leaving this as an exception permanently results in kind of a strange user experience though.  I would tend toward letting the user decide if he wants to see edges and vertices, even if it doesn't make sense.  I find pretty senselessness preferable to runtime errors.  But if you do choose to leave it this way, please mention in the documentation that it's not implemented.